### PR TITLE
"From Cute to Brute" Secret Lair deck

### PR DESCRIPTION
**Pull request by davidmartos96**
Hello, here is a work in progress on the From Cute to Brute deck.
The Commander and 4 cards are the new Secret Lair printings. I specified the set and number for those.
The remaining 95 cards are stamped in the corner with the Planeswalker icon, same as the other Secret Lair deck, "Heads I Win, Tails you Lose" (PHED) https://scryfall.com/sets/phed

Since these printings are not yet available on Scryfall or MTGJSON I used the sld folder, but that should probably be changed once we have a particular set code for the deck, so that the printings resolve correctly. 

https://magic.wizards.com/en/news/announcements/commander-returns-to-secret-lair-with-from-cute-to-brute
https://secretlair.wizards.com/us/product/819075/secret-lair-commander-deck-from-cute-to-brute
https://mtg.fandom.com/wiki/Secret_Lair_Commander_Deck:_From_Cute_to_Brute

### CODE REVIEW

File: data/cmd/sld/From Cute to Brute.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// NAME: From Cute to Brute
+// SOURCE: https://mtg.fandom.com/wiki/Secret_Lair_Commander_Deck:_From_Cute_to_Brute
+// DATE: 2023-05-08
+COMMANDER: 1 Esika, God of the Tree // The Prismatic Bridge [SLD:1155]
```
Comment (axxroytovu) [inline data/cmd/sld/From Cute to Brute.txt:4]: Numbering with double faced cards should only associate the front face. eg:
`COMMANDER: 1 Esika, God of the Tree [SLD:1155a]`

**Review by axxroytovu — CHANGES_REQUESTED** [review]

**Comment by axxroytovu** [comment]
@taw this is breaking because the SLD "set" release date doesn't match up with the deck release date, and cards like Dennick, Pious Apprentice are breaking. 

`deck_printing_resolver.rb:124:in 'resolve_set': All printings of Dennick, Pious Apprentice from the future`

Can we adjust the deck parser to use the recorded deck release date if it exists? Right now it just uses `release_date(deck_set_code)`.

File: data/cmd/sld/From Cute to Brute.txt (hunk 1 above)
Comment (axxroytovu) [reply data/cmd/sld/From Cute to Brute.txt:4]: This only applies to the numbered cards, the non-numbered double faced cards are fine.

Comment (taw) [reply data/cmd/sld/From Cute to Brute.txt:4]: I added a step to deck indexer that can handle both [SLD:1155] and [SLD:1155a]

**Comment by taw** [comment]
@axxroytovu Date handling can now deal with this.

File: data/cmd/sld/From Cute to Brute.txt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+1 Cragcrown Pathway // Timbercrown Pathway
+1 Needleverge Pathway // Pillarverge Pathway
+1 Riverglide Pathway // Lavaglide Pathway
+1 Valakut Awakening // Valakut Stoneforge
+1 Bala Ged Recovery // Bala Ged Sanctuary
+1 Clearwater Pathway // Murkwater Pathway
+1 Hagra Mauling // Hagra Broodpit
+1 Ondu Inversion // Ondu Skyruins
+1 Sandstone Oracle
+1 Sphinx of the Second Sun
```
Comment (axxroytovu) [inline data/cmd/sld/From Cute to Brute.txt:58]: This specific card can't resolve with the current search rules. Should add [CMR:99] for now.

Comment (davidmartos96) [reply data/cmd/sld/From Cute to Brute.txt:58]: @taw Do you want to merge and index the deck anyway? Or do you prefer to wait for the correct printings?

Comment (davidmartos96) [reply data/cmd/sld/From Cute to Brute.txt:58]: We can wait a bit. I just found out that the other 95 cards are now in Scryfall. They will be fetched by MTGJSON in the next update and we can index them here afterwards.
https://scryfall.com/sets/pctb

**Pull request merged**